1.7/administration/installing/custom/advanced.md

```diff
@@ -62,7 +62,7 @@ The DC/OS installation creates these folders:
 
     **Important:** The IP address of a node must not change after DC/OS is installed on the node. For example, the IP address must not change when a node is rebooted or if the DHCP lease is renewed. If the IP address of a node does change, the node must be wiped and reinstalled.
 
-    **Important:** The IP detection scripts must begin with `#!/bin/sh` or `#!/usr/bin/env bash` as shown in the examples below. Do not add any comments before that.
+    **Important:** The IP detection script needs to be `UTF-8` encoded and have a valid [Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) line as shown in the examples below.
```

Comment:
@joel-hamill I believe that the url should be this 
[shebang](https://en.wikipedia.org/wiki/Shebang_%28Unix%29) == `[shebang](https://en.wikipedia.org/wiki/Shebang_%28Unix%29)`

using ur doc updates i would get 

[Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix&#41;) == `[Shebang](https://en.wikipedia.org/wiki/Shebang_(Unix&#41;)`

this will then work in both git markdown and html markdown.